chaosaws/ec2/actions.py

```diff
@@ -17,3 +17,54 @@ def stop_instance(instance_id: str, force: bool = False,
     """
     client = aws_client('ec2', configuration, secrets)
     return client.stop_instances(InstanceIds=[instance_id], Force=force)
+
+
+def stop_random_instance(force: bool=False,
```

Comment:
Note that we lint with a space around the assign sign: `force : bool = False` :)